backend/controllers/commentController.js

```diff
@@ -0,0 +1,63 @@
+import mongoose from "mongoose"
+import Post from "../models/postModel.js"
+import Comment from "../models/commentModel.js"
+
+async function addComment(req, res) {
+    const {
+        content,
+        postId
+    } = req.body
+    const comment = {
+        content,
+        postId,         //check if needs auth changes
+        author: req.user.userName,
+        userId: req.user.id,
+        family: req.user.family,
+    }
+    const newComment = new Comment(comment)
+    try {
+        let mdb = await newComment.save()
+        res.json(mdb)
+    } catch (error) {
+        res.json(error)
+    }
+}
+
+async function editComment(req, res) {
```

Comment:
Need to check that the user is editing his own comment and not another user's comment